### ISSUE TITLE
Error when trying to start

### ISSUE THREAD

**Issue by micgro42** [gingivitis/i3-pomodoro#1]
There is an error on `npm run`:

Error-Message:
```
Server bound on port 41234
{ Error: ENOENT: no such file or directory, open '/home/michael/tmp/pomodoro'
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/michael/tmp/pomodoro' }
Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379
    at Object.exports._errnoException (util.js:1033:11)
    at exports._exceptionWithHostPort (util.js:1056:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1099:14)
/home/michael/sources/i3-pomodoro/lib/pomodoro.js:51
        let period = this.periods[0]
                                 ^

TypeError: Cannot read property '0' of undefined
    at Pomodoro.pause (/home/michael/sources/i3-pomodoro/lib/pomodoro.js:51:34)
    at exitHandler (/home/michael/sources/i3-pomodoro/main.js:20:14)
    at emitOne (events.js:96:13)
    at process.emit (events.js:191:7)
    at process._fatalException (bootstrap_node.js:299:26)
error Command failed with exit code 7.
```

**Comment by parthsharma1996** [comment]
Same problem

**Comment by roymand** [comment]
Me too, has anyone found a way to make it work? I see the block in i3blocks, but nothing happens if I click on it, and running it from the command line with npm start gives this error.

**Comment by neochar** [comment]
You need to have redis service running.
For archlinux I did:
```
sudo pacman -S redis
sudo systemctl start redis.service
```

If you want redis to be automatically started after login run
```
sudo systemctl enable redis.service
```

**Comment by odontogenic** [comment]
tldr install & run redis... thanks @neochar!
